test/hotspot/jtreg/serviceability/sa/sadebugd/SADebugDTest.java

```diff
@@ -71,6 +71,17 @@ private static void runTests() throws Exception {
         }
     }
 
+    private static boolean checkOutput(final String line, final int rmiPort) {
+        boolean useRmiPort = rmiPort != -1;
```

Comment:
I think rather than recomputing useRmiPort, which the caller already knows, you should instead just pass it in as an argument.